src/include/dawg/indel.h

```
@@ -46,17 +47,23 @@ class indel_model {
         };
 
         if(max_size > std::numeric_limits<boost::uint32_t>::max()) {
-            return DAWG_ERROR("maximum indel size is out of range");
+	    std::error_code ec = dawg_error::invalid_value;
```

You should be able to `throw dawg_error::invalid_value`